src/Arcus.Observability.Tests.Unit/Correlation/DefaultCorrelationInfoAccessorTests.cs

```diff
@@ -19,15 +19,15 @@ public async Task SetCorrelationInfo_Twice_UsesMostRecentValue()
             await SetCorrelationInfo(firstOperationId, transactionId);
 
             // Act
-            await SetCorrelationInfo(secondOperationId, transactionId);
+            SetCorrelationInfo(secondOperationId, transactionId);
 
             // Assert
             CorrelationInfo correlationInfo = DefaultCorrelationInfoAccessor.Instance.GetCorrelationInfo();
             Assert.Equal(secondOperationId, correlationInfo.OperationId);
             Assert.Equal(transactionId, correlationInfo.TransactionId);
         }
 
-        private async Task SetCorrelationInfo(string operationId, string transactionId)
+        private void SetCorrelationInfo(string operationId, string transactionId)
```

Comment:
Ah, no we can't do that. We should make this `async`  bc that was the cause why a previous bug was reported and this test checks that.

Reply:
I don't understand; what exactly are we testing ?  This method is marked as async, but doesn't do anything async ?

Reply:
In a previous version, the `DefaultCorrelationAccessor` was not fully implemented correctly. It was not using the most recent set correlation information. This was caused due to asynchronous calls towards the accessor. The setup of the test was to verify that this is now fixed, and that asynchronous calls towards this accessor (and setting the correlation information) still results in the most recent set correlation information. 

If you want, you can add a task delay to make it more async, but we still need to make sure this is async, so it has to be reverted.

Reply:
I will revert it, but to be honest, I don't think it will matter much. 
Since there are no awaitable calls, the method will just run synchronously.

Reply:
Also, `DefaultCorrelationInfoAccessor.Instance` is deprecated ?

Reply:
Yeah, I think it's mostly about context switching, not about asynchronous call explicitly.
Thx!
Yeah, indeed, so we probably will be able to remove the test once it's removed.